[DOC] corrected a word

**Provide a link to that doc page:**
https://github.com/211820252/fluid/blob/master/pkg/ddc/goosefs/transform_api_gateway_test.go
**What is the defect and your suggestions on improvement:**
pkg\ddc\goosefs\transform_api_gateway_test.go:85:34: corrected "beacuse" to "because"

--- a/pkg/ddc/goosefs/transform_api_gateway_test.go
+++ b/pkg/ddc/goosefs/transform_api_gateway_test.go
@@ -82,7 +82,7 @@ func TestTransformAPIGateway(t *testing.T) {
 				t.Errorf("should return err if it's possible to lead to nil pointer")
 			}
 		} else if test.runtime.Spec.APIGateway.Enabled != test.value.APIGateway.Enabled {
-			t.Errorf("testcase cannot paas beacuse of wrong result,%t != %t",
+			t.Errorf("testcase cannot paas because of wrong result,%t != %t",
 				test.runtime.Spec.APIGateway.Enabled,
 				test.value.APIGateway.Enabled)
 		}

